File: Semester 4/Programming and Projection Enironments/mpp-proiect-java/Proiect_MPP/src/main/java/root/proiect_mpp/repositories/airports/IAirportRepository.java
package root.proiect_mpp.repositories.airports;

import root.proiect_mpp.domain.Airport;
import root.proiect_mpp.repositories.Repository;

import java.util.Collection;

public interface IAirportRepository extends Repository<Airport, Integer> {
    Collection<Airport> getAirportAfterName(String name);
    Collection<Airport> getAirportsInCity(String cityName);
}

File: Semester 4/Programming and Projection Enironments/mpp-proiect-csharp/Proiect_MPP/Proiect_MPP/repository/airports/IAirportRepository.cs
using Proiect_MPP.domain;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proiect_MPP.repository.airports
{
    public interface IAirportRepository : Repository<Airport, int>
    {
        List<Airport> getAirportAfterName(string name);
        List<Airport> getAirportsInCity(string cityName);
    }
}
